lib/travis/api/v3/renderer/build.rb

```diff
@@ -4,17 +4,17 @@ module Travis::API::V3
   class Renderer::Build < Renderer::ModelRenderer
     representation(:minimal,  :id, :number, :state, :duration, :event_type, :previous_state, :pull_request_title, :pull_request_number, :started_at, :finished_at)
     representation(:standard, *representations[:minimal], :repository, :branch, :commit, :jobs)
-    representation(:active, *representations[:minimal], :repository, :branch, :commit, :jobs)
+    representation(:active, *representations[:standard])
 
     def jobs
-      if include_full_jobs?
-        return representation?(:active) ? model.active_jobs : model.jobs
-      end
-      model.job_ids.map { |id| job(id) }
+      return model.active_jobs if include_full_jobs? && representation?(:active)
+      return model.jobs if include_full_jobs?
+      return model.job_ids.map { |id| job(id) } unless representation?(:active)
+      model.active_jobs.map{ |j| job(j.id) }
```

Comment:
the minimal representation was returning jobs that were not active. This is hopefully a strange quirk of staging that we have builds in 'created' and 'started' that do not have any active_jobs. Was this intentional? It was producing confusing results because for the same owner, when I said '?include=build.jobs' I got no jobs, but without the include I was getting jobs.